components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp

```diff
@@ -205,11 +205,8 @@ function disableAssertionSignature(chkbx) {
             : false;
 }
 function disableAttributeProfile(chkbx) {
-    document.addServiceProvider.claim.disabled = (chkbx.checked) ? false
-            : true;
-    document.addServiceProvider.addClaims.disabled = (chkbx.checked) ? false
-            : true;
-
+    if (!(chkbx.checked))document.addServiceProvider.enableDefaultAttributeProfile.checked = false;
+    document.addServiceProvider.enableDefaultAttributeProfile.value = (chkbx.checked) ? true : false;
```

Comment:
this is same as document.addServiceProvider.enableDefaultAttributeProfile.value = chkbx.checked ?